backend/compact-connect/tests/unit/test_app.py

```diff
@@ -95,6 +95,7 @@ def test_synth_sandbox_no_domain(self):
         self._check_no_annotations(app.sandbox_stage.persistent_stack)
         self._check_no_annotations(app.sandbox_stage.ui_stack)
         self._check_no_annotations(app.sandbox_stage.api_stack)
+        self._check_no_annotations(app.sandbox_stage.ingest_stack)
```

Comment:
This related to this PR or just caught something from a past sprint?

Reply:
This change is not related to this PR, but it is a change that should have already been merged in as part of the domain name PR. We had a bit of a knot tied in our git history as we converted from the per-sprint PRs to per-feature PRs. I'm guessing I somehow backed this change back out of my PR when we were untying that knot. Can we keep this in here so it gets corrected with this merge?

Reply:
Got it. Sounds good!